src/index.php

```diff
@@ -1,83 +1,50 @@
 <?php
 
-namespace Islandora\CollectionService;
+namespace Islandora\PDX;
 
 require_once __DIR__.'/../vendor/autoload.php';
 
 use Silex\Application;
-use Islandora\Chullo\Uuid\UuidGenerator;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\HttpKernelInterface;
 use Psr\Http\Message\ResponseInterface;
 use Silex\Provider\TwigServiceProvider;
-use Islandora\ResourceService\Provider\ResourceServiceProvider;
-use Islandora\CollectionService\Provider\CollectionServiceProvider;
-use Islandora\TransactionService\Provider\TransactionServiceProvider;
+use Islandora\PDX\CollectionService\Provider\CollectionServiceProvider;
 
 date_default_timezone_set('UTC');
 
 $app = new Application();
 
 $app['debug'] = true;
+$app['islandora.BasePath'] = __DIR__;
 $app->register(new \Silex\Provider\ServiceControllerServiceProvider());
-$app->register(
-    new \Silex\Provider\TwigServiceProvider(),
-    array(
-      'twig.path' => __DIR__.'/../templates',
-    )
-);
-
-$islandoraResourceServiceProvider = new ResourceServiceProvider;
-//Registers Resource Service and defines current app's path for config context
-$app->register(
-    $islandoraResourceServiceProvider,
-    array(
-      'islandora.BasePath' => __DIR__,
-    )
-);
-$app->mount("/islandora", $islandoraResourceServiceProvider);
-
-$islandoraTransactionService = new TransactionServiceProvider;
+// TODO: Not register all template directories right now.
+$app->register(new \Silex\Provider\TwigServiceProvider(), array(
+  'twig.path' => array(
+    __DIR__ . 'CollectionService/templates',
+  ),
+));
 
-$app->register($islandoraTransactionService);
-$app->mount("/islandora", $islandoraTransactionService);
+$islandoraCollectionServiceProvider = new CollectionServiceProvider;
 
-$islandoraCollectionService = new CollectionServiceProvider;
-
-$app->register(
-    $islandoraCollectionService,
-    array(
-      'UuidGenerator' => new UuidGenerator(),
-    )
-);
```

Comment:
This register command should remain, but you can change the variable name to match `$islandoraCollectionServiceProvider`